### PR TITLE
Prepare for slimming RA weapon yamls Part 1 - explosions

### PR DESCRIPTION
**Pull request by reaperrr**
This should be pretty uncontroversial.
I think we should get this into the next playtest for that very reason.

Note: I HIGHLY recommend reviewing commit by commit, not the whole diff at once!

### CODE REVIEW

**Comment by reaperrr** [comment]
@SoScared @Micr0Bit and any other players around, any suggestions regarding the last commit (Vulcan)?

I know such changes are tricky for balancing, but I also think this change is overdue, the Vulcan's current setup is just horrible to tweak.

**Comment by reaperrr** [comment]
Updated, dropped the last commit for now due to lack of feedback.

**Comment by SoScared** [comment]
Sorry for the absence, been a busy couple of weeks. Can I still find the Vulcan code/commit somewhere? I could store it and sneak it in with a playtest map/build at a later date.

**Comment by reaperrr** [comment]
> Sorry for the absence, been a busy couple of weeks. Can I still find the Vulcan code/commit somewhere? I could store it and sneak it in with a playtest map/build at a later date.

Sure, here you go (last commit): https://github.com/reaperrr/OpenRA/tree/RA-vulcan

**Comment by Micr0Bit** [comment]
@reaperrr : it will collide with their idea to make all the allies defense structures more expensive (including the pillbox for 600 instead of 400) ... which i was never a fan of

so a pillbox-nerf (which this pr in its current state contains) will mostly make things complicated to justify .. so it either gets properly adjusted or maybe other (balance)changes that are in the pipeline need to get cut out 

"sneaking" (as is) something in, is most likely the wrong way 

i did test the pillbox , and im fine with it ... but combined with a pricebuff (+ higher buildtime) players probably wont accept this (maybe ss+fans)

im all for a cleanup of this yaml-part though ... so :+1: from me after the "experts" found time to take a look and adjust it

**Comment by reaperrr** [comment]
Well, if we do change it, I guess it's safer to do that later in a separate PR, from a yaml-complexity perspective it's just a "nice-to-have", not mandatory.

**Comment by FrameLimiter** [comment]
I just tested this as well and noticed a couple things:
1.) Moving targets get tracked way better so it's much more difficult to out run the pillbox.
2.) The busts of 3 give it the same issue the jeep/apc have where it doesn't reload a full 3 bursts if it only uses 1 or 2. (couldn't this be fixed by using AmmoPools?)
3.) The reload delay is 30 but with that burst delay of 2 the total reload delay is 34. (maybe set reload delay to 26 to compensate for the burst delays?
4.) Medics are almost impossible to kill. This is because they heal between each burst interval.

**Review by pchote — APPROVED** [review]
I've been through the commits and agree that this is uncontroversial.  The CrateNuke changes aren't going to have any noticable effect ingame.

**Comment by reaperrr** [comment]
Updated, I split this into 4 parts to hopefully reduce the "reviewers put off by large number of commits/changed yaml lines" effect.

No other changes, so @pchote's 👍 should still be valid on all 4 of them.

File: mods/ra/weapons/explosions.yaml [hunk 1]
```diff
@@ -243,34 +243,34 @@ CrateNuke:
 	Warhead@3Eff_impact: CreateEffect
 		Explosions: nuke
 		ImpactSounds: kaboom1.aud
-	Warhead@4Dam_areanuke: SpreadDamage
+	Warhead@4Dam_areanuke1: SpreadDamage
```
Comment (abcdefg30) [inline mods/ra/weapons/explosions.yaml]: Was there a special reasoning behind naming this `areanuke1`? (I don't see a second areanuke.)

**Comment by abcdefg30** [comment]
Looks good to me otherwise. :+1: / :white_check_mark: when the question is resolved.

File: mods/ra/weapons/explosions.yaml (hunk 1 above)
Comment (reaperrr) [inline mods/ra/weapons/explosions.yaml]: I'm considering (not entirely sure yet, due to the difference in total warhead count and relatively low yaml line savings) to make MiniNuke inherit CrateNuke, so I renamed this now so I won't have to touch CrateNuke if/when I do that.

**Pull request merged**